scripts/test-coverage-xml.sh

```diff
@@ -14,10 +14,10 @@ export ZENML_ANALYTICS_OPT_IN=false
 export EVIDENTLY_DISABLE_TELEMETRY=1
 
 if [ -n "$1" ]; then
-    coverage run -m pytest $TEST_SRC --color=yes
+    coverage run -m pytest $TEST_SRC --color=yes -n auto -vv
```

Comment:
Slightest nit: I had to look up both `-n auto` and why we added `-vv` (to enable pytest-clarity), so it might be nice to add a short explanation here. Feel free to ignore if you/others don't think it's necessary

Reply:
Do you mean a comment in the script?

Reply:
Yeah just a one-line comment inside the script that says that it's running on the amount of cpu cores